### PR TITLE
Add support for a pile of trivial extensions.

### PR DESCRIPTION
**Pull request by AWoloszyn**
These extensions are trivial, so I added them all together.
Otherwise you end up with merge conflicts for each.

Newly supported extensions:
- VK_EXT_depth_range_unrestricted
- VK_EXT_pci_bus_info
- VK_EXT_shader_stencil_export
- VK_EXT_shader_subgroup_ballot
- VK_EXT_shader_subgroup_vote
- VK_EXT_shader_viewport_index_layer
- SPV_KHR_storage_buffer_storage_class
- VK_KHR_8bit_storage
- VK_AMD_shader_core_properties
- VK_GOOGLE_decorate_string
- VK_GOOGLE_hlsl_functionality1
- VK_KHR_spirv_1_4

### CODE REVIEW

**Comment by pau-baiget** [comment]
I know it can be seen in the code changes, but would you mind adding the list of newly supported extensions to this PR's description?

**Comment by pau-baiget** [comment]
> I know it can be seen in the code changes, but would you mind adding the list of newly supported extensions to this PR's description?

Nevermind, updated.

**Comment by AWoloszyn** [comment]
Thanks!

File: gapis/api/vulkan/vulkan.api [hunk 1]
```diff
@@ -256,6 +260,18 @@ sub ref!ExtensionSet supportedDeviceExtensions() {
   supported.ExtensionNames["VK_KHR_descriptor_update_template"] = true
   supported.ExtensionNames["VK_KHR_device_group"] = true
   supported.ExtensionNames["VK_GOOGLE_display_timing"] = true
+  supported.ExtensionNames["VK_EXT_depth_range_unrestricted"] = true
+  supported.ExtensionNames["VK_EXT_pci_bus_info"] = true
+  supported.ExtensionNames["VK_EXT_shader_stencil_export"] = true
+  supported.ExtensionNames["VK_EXT_shader_subgroup_ballot"] = true
+  supported.ExtensionNames["VK_EXT_shader_subgroup_vote"] = true
+  supported.ExtensionNames["VK_EXT_shader_viewport_index_layer"] = true
+  supported.ExtensionNames["SPV_KHR_storage_buffer_storage_class"] = true
```
Comment (chrisforbes) [inline gapis/api/vulkan/vulkan.api]: This is wrong. The Vulkan extension is `VK_KHR_storage_buffer_storage_class`.

Comment (chrisforbes) [reply gapis/api/vulkan/vulkan.api]: (SPV_KHR_storage_buffer_storage_class is the SPIRV part)

Comment (AWoloszyn) [reply gapis/api/vulkan/vulkan.api]: thanks!

**Pull request merged**